src/homepage/_posts/2019-02-12-james-3.3.0.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title:  "Apache James Server 3.3.0"
+date:   2019-02-12 12:13:22 +0700
+categories: james update
+---
+
+The Apache James developers are pleased to announce James server 3.3.0 release.
+
+Here are some points we worked on:
+
+### Added
+
+- Metrics for BlobStore
+- New Guice product using Cassandra RabbitMQ ElasticSearch, OpenStack Swift and optional LDAP dependency (experimental)
+- JPA SMTP dockerFile contributed by [matzepan](https://github.com/matzepan)
+- Listing healthchecks, thanks to [Madhu Bhat](https://github.com/kratostaine)
+- Configuring the ElasticSearch clusterName
+- Logging and Metrics now supports Elasticsearch 6 (previously only Elasticsearch 2 was supported)
+- Implementation of the RabbitMQ EventBus
+- DeadLetter APIs and memory implementation for storing events that failed delivery
+- RecipientRewriteTable Aliases and associated WebAdmin routes
+
+### Fixed
+
+- MAILBOX-350 Potential invalid UID <-> MSN mapping upon IMAP COPY
+- Possibility to better zoom in Grafana boards
+- default ElasticSearch shards & replica configured values
+- Move & copy batch sizes are now loaded from configuration
+
+### Changed
```

Comment:
how about adding one more item of Reactor migration?

Reply:
Sounds a good idea